omni/omni/omni_v0.h

```diff
@@ -47,7 +47,7 @@ typedef struct {
 StaticAssertDecl(sizeof(omni_magic) <= UINT16_MAX, "omni_magic should fit into 16 bits");
 
 #define OMNI_INTERFACE_VERSION 0
-#define OMNI_INTERFACE_REVISION 5
+#define OMNI_INTERFACE_REVISION 6
```

Comment:
I just realized there are two types of versions in omni:
1. `x.y.z` semver version for extension
2. `OMNI_INTERFACE_REVISION` for omni interface
Isn't it confusing because we can introducing breaking changes by bumping the second type of version without reflecting it in the extension version ?

Reply:
Yes, indeed there are two. Interface versioning implies only the shape of the interface. Works this way:

< OMNI_INTERFACE_VERSION ><OMNI_INTERFACE_REVISION> (where it is suggested to translate OMNI_INTERFACE_REVISION to a letter in the alphabet to make sure people don't treat it as a SemVer; so it'd be 1A, 2B, etc.)

The idea is that 1B can only augment 1A but can't break compatibility, where as 2A can break anything from 1*.

The only exception is VERSION 0: in 0A..Z where we can break anything.